types/responses.ts

```diff
@@ -124,6 +159,10 @@ export interface GenerateContentCandidate {
   finishMessage?: string;
   safetyRatings?: SafetyRating[];
   citationMetadata?: CitationMetadata;
+  /** Average log probability score of the candidate. */
+  avgLogProbs?: number;
```

Comment:
since it's `avg_logprobs` in the proto, the capitalization should translate to `avgLogprobs` here.